fecfile_validate_python/pyproject.toml 

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
```

Comment:
I noticed there are a few files without newlines at the end - I think there's a VS code setting that will automatically add one for you.

Reply:
done